tst/EnergyPlus/unit/AirTerminalSingleDuctMixer.unit.cc

```diff
@@ -7896,14 +7896,14 @@ TEST_F(EnergyPlusFixture, AirTerminalSingleDuctMixer_SimFCU_ATMInletSideTest)
     SecondaryAirMassFlowRate = state->dataLoopNodes->Node(thisFanCoil.AirInNode).MassFlowRate - PrimaryAirMassFlowRate;
     // check results in cooling mode operation
     EXPECT_NEAR(QZnReq, QUnitOut, 5.0);
-    EXPECT_NEAR(thisFanCoil.PLR, 0.78843, 0.00001);
+    EXPECT_NEAR(thisFanCoil.PLR, 0.76235, 0.00001); // Was 0.78843 
     // check mass flow rates
     EXPECT_NEAR(PrimaryAirMassFlowRate, 0.2, 0.000001);
-    EXPECT_NEAR(SecondaryAirMassFlowRate, 0.369714, 0.000001);
+    EXPECT_NEAR(SecondaryAirMassFlowRate, 0.350865, 0.000001);
     EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.AirInNode).MassFlowRate, thisFan.InletAirMassFlowRate, 0.000001);
     EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.ATMixerPriNode).MassFlowRate, 0.2, 0.0001);
-    EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.ATMixerSecNode).MassFlowRate, 0.369714, 0.000001);
-    EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.ATMixerOutNode).MassFlowRate, 0.569714, 0.000001);
+    EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.ATMixerSecNode).MassFlowRate, 0.350865, 0.000001); // Was 0.369714
+    EXPECT_NEAR(state->dataLoopNodes->Node(thisFanCoil.ATMixerOutNode).MassFlowRate, 0.550865, 0.000001); // Was 0.569714
```

Comment:
Yeah, this is the same thing I see. I still can't figure out why this unit test fails now and not last year when that 1 line (line 515 in FanCoilUnits) was commented out (#9932 on Apr 3, 2023). It makes sense that the cooling coil is now larger (by 496.3 W of fan power) and the PLR is lower which changes the air flow numbers. The example file FanCoilAutoSize skips sizing the first time through but calls the fan (also skips sizing), which sets the fanIndex so there is no change in results when sizing finally does occur. That explains the example files but not this unit test.